train/net_utils.py

```diff
@@ -15,6 +15,7 @@
 
 
 def mask_illegal_moves(target, output):
+    output = tf.cast(output, tf.float32)

```

Comment:
what does this do? 
theoretically the output doesn't need a cast

Reply:
No idea, saw it in Lc0